graphql_compiler/compiler/emit_sql.py

```diff
@@ -169,9 +215,10 @@ def __init__(self, outer_vertex_table, join_descriptor):
         #  from_table: the table on the left side of the join
         #  to_table: the table on the right side of the join
         self._join_info = []
-
         self._outputs = []  # output columns for fold
 
+        self._is_mssql2014 = is_mssql2014  # indicate results should be compatible with MSSQL2014
```

Comment:
This function has a bunch of unnecessary formatting changes, which should be reverted. `is_mssql2014` needs an entry in the args section.